.github/CODEOWNERS

```diff
@@ -8,5 +8,6 @@
 # global owners are only requested if there isn't a more specific
 # codeowner specified below. For this reason, the global codeowners
 # are often repeated in package-level definitions.
-* @liamsi @MSevey @evan-forbes @renaynay @Wondertan @tzdybal @Bidon15
+* @liamsi @MSevey
+/.github/ @celestiaorg/devops
```

Comment:
@MSevey are we sure we have such a team in the org? Seems like not 😓

Reply:
If yes according to this https://github.com/orgs/celestiaorg/teams/devops

please let me know how I can grant necessary access rights for this repo

Reply:
I gave devops maintain perms.